code/game/objects/items/weapons/tools/pickaxe.dm

```diff
@@ -55,6 +55,10 @@
 	use_fuel_cost = 0.15
 	max_fuel = 120
 
+/obj/item/weapon/tool/pickaxe/diamonddrill/nofuel
```

Comment:
this feels kinda hacky, but it'll do for now. The drill still uses power right?

Also change the name and path to something application specific like /diamonddrill/rig